### PR TITLE
fix(frontend): balance precision increase

### PR DESCRIPTION
**Pull request by divyam1705**
- increases the precision for token balances

### CODE REVIEW

File: packages/frontend/src/utils/format.ts [hunk 1]
```diff
@@ -13,6 +13,9 @@ export function formatPercentage(amount?: number | null): string {
 
 export function formatTokenAmount(amount?: number | null): string {
     if (amount && amount < 0.0001) return "<0.0001";
+    if (amount && amount < 1) {
+        return `${numeral(amount).format("0,0.0000")}`;
```
Comment (guerrap) [inline packages/frontend/src/utils/format.ts:17]: I think we can have 2 min decimals and 2 optional, so `0,0.00[00]`, would that be ok?

Comment (divyam1705) [reply packages/frontend/src/utils/format.ts:17]: How about this?

```
export function formatTokenAmount(amount?: number | null): string {
    if (amount && amount < 0.0001) return "<0.0001";
    return `${numeral(amount).format("0,0.0[000]a")}`;
}

```

Comment (guerrap) [reply packages/frontend/src/utils/format.ts]: Could work, but the `a` should be optional, there's a change on develop for this that's causing the conflicts.

Comment (divyam1705) [reply packages/frontend/src/utils/format.ts]: Merged✅

**Review by luzzif — APPROVED** [review]

**Review by guerrap — APPROVED** [review]

**Pull request merged**